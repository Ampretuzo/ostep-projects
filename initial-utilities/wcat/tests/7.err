tests/7c.in: No such file or directory
