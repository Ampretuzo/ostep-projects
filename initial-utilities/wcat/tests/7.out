this one has stuff in it
this one does too
this one does as well and should get printed too
